### DISCUSSION TITLE
BaGetter Icon

### DISCUSSION THREAD

**Discussion by seriouz** [bagetter/BaGetter#119] [General]
I've created an icon for BaGetter.

![image](https://github.com/bagetter/BaGetter/assets/4722613/9685762b-242f-41eb-bfb7-ee4e3dc18551)

**Comment by joestr** [comment]
I think there is a lack of baguette in the icon. I just imagined a rocket ship in the form a baguette?

**Reply by joestr** [reply]
![_d79cde88-2c1a-473c-a5e5-0b765ad22744](https://github.com/bagetter/BaGetter/assets/20691668/8216cddb-0185-4020-9969-8a03e942b42e)

Generated with copilot

**Reply by seriouz** [reply]
i like the idea!

**Reply by Regenhardt** [reply]
I like the concept. I just feel like it might be too complex for an icon.

**Reply by Regenhardt** [reply]
@joestr what was your prompt for this?

**Reply by joestr** [reply]
https://copilot.microsoft.com/sl/eFWHiC1nxtI

**Reply by seriouz** [reply]
I got these:

![image](https://github.com/bagetter/BaGetter/assets/4722613/7bcff414-0c17-4cfc-b283-87d11d9657e4)

Maybe the portholes could be arranged like the NuGet white dots.

**Comment by FroggieFrog** [comment]
I think that maybe that new thing "AI image creation" 😀 could be helpful. Maybe.
Just a quick test:
![grafik](https://github.com/bagetter/BaGetter/assets/6222752/c772a828-172c-4a3e-8a36-36c3638b42a9)

**Reply by joestr** [reply]
This could actually be useful.

**Reply by seriouz** [reply]
I could try to simplify such basket. But i won't guarantee that it still looks good. Directly copying such picture could lead to violation of rights of use.

**Comment by FroggieFrog** [comment]
Just to add one thing:
I don't know if it is allowed to "use" the NuGet logo in projects that are not related to Microsoft. I'm not a lawyer.
